torch/distributed/_tensor/_dispatch.py

```diff
@@ -113,9 +116,21 @@ def dispatch(
 
         # extract local tensor and sharding infos to a OpInfo
         op_info = self.unwrap_to_op_info(op_call, args, kwargs)
+        if logger.isEnabledFor(logging.DEBUG):
```

Comment:
I wonder if this if check is for perf reason?

Reply:
Yes, but if we don't need to unwrap, then it is not necessary.